internal/caller/telephone.go

```diff
@@ -0,0 +1,62 @@
+package caller
```

Comment:
Please move this file to `pkg/telephone/telephone` and adjust the package name accordingly so that this can be importable by other users.